logging-infrastructure/README.md

```diff
@@ -0,0 +1,62 @@
+# Logging infrastructure
+
+As a Giant Swarm engineer, I want to be able to access a history of logs for components managing our platform in order to both investigate ongoing operational issues and provide details for incident reports.
+
+## Context
+
+Atlas started working on a logging infrastructure implementation using [Loki](https://grafana.com/docs/loki/latest/).
+So far we worked on a POC towards having a centralized setup, where a central Loki instance hosted by GiantSwarm ingests logs from our installations.
+There were some recent discussions where we learned about other requirements leaning towards a distributed setup, here are the pro and cons details of those discussions.
+
+### Distributed setup
+
+What we call distributed setup, is when there is 1 Loki instance hosted on each installation.
+So logs are stored and accessible on a per installation basis.
+
+#### Pros
+
+- Satisfy customer requirement towards data isolation (i.e. legal, or data privacy requirements)
```

Comment:
FYI, I think this is the only real reason why we have to go distributed

Reply:
Yes, that's what I mention later on in the conclusion.